packages/docs/verite/patterns/travel-rule.md

```diff
@@ -2,53 +2,104 @@
 
 <img src="/img/docs/travel_rule.png" alt="Travel Rule" width="800"/>
 
 ### Part 1: Discovery and Proving Control of Blockchain Addresses
 
+Crucial to the design of our system is some kind of bulletin board system which can be conceived of as a miniature "mempool", i.e. a smart contract that hosts ephemeral messages posted publically for the express purpose of being responded to privately by self-authenticating parties.
+
+Before an intended/proposed transaction is broadcast, the sending and receiving FIs use this chain-agnostic "bulletin board" to publish and observe messages establishing and requesting proof of address control. **These messages contain no customer data and no specific transaction data other than the *recipient* address and sending *custodian's* identity.** 
+
+Since the transaction amount is not included, observers are unable to determine information about the planned transaction other than its intended origin and destination (this is less information than is available in the EVM-wide mempool, for example, which broadcasts entire transaction ready to be added to blocks). This allows counterparty endpoints to find and connect to one another privately **before** exchanging counterparty information, and the mutual exchange of proof-of-control credentials for both addresses prevent any other party from spoofing the originator or beneficiary.
+
 1. Alice, the originator, initiates a transfer with her financial institution, which operates a hosted crypto wallet on her behalf.
 2. Alice's FI forms a message (referred to here as "M1"), containing:
-   - Proof of control of the originating address (formatted as a Verifiable Credential)
-   - Request for proof of control of the beneficiary address (formatted as a Presentation Request)
-   - A secure callback URL endpoint for replies to the Presentation Request
-3. Alice's FI publishes M1 to the decentralized mem pool.
-4. Bob's financial institution observes M1 in the mem pool and determines that the recipient address is in its custody, and it verifies the originator's VC.
-5. Bob's financial institution constructs message "M2" which is a VC proving control over the beneficiary address.
-6. Bob's FI transmits M2 privately to Alice's FI using the callback URL specified in M1.
-7. Alice's FI receives and verifies the VC in M2, leaving both Alice's FI and Bob's FI (a) certain that each controls the designated addresses and (b) with a private secure network connection established through the specified callback URL.
+   - Request for proof of control of the beneficiary address (e.g. formatted as a Presentation Request or [CACAO receipt][CACAO] of a [SIWX message][])
+   - One or more identifiers for querying and verifying the identity of the custodian initiating the message
+   - A secure callback URL endpoint for replies (e.g. a Presentation Exchange endpoint for receiving Verifiable Presentations)
+4. Additionally, an FI message should contain, if in "**Fast mode**" :
```

Comment:
"FI message" ? I think it's a typo for "M1 message"

Reply:
good catch